sdks/java/core/src/test/java/org/apache/beam/sdk/transforms/reflect/DoFnInvokersTest.java

```diff
@@ -92,6 +92,7 @@
 @RunWith(JUnit4.class)
 @SuppressWarnings({
   "rawtypes", // TODO(https://issues.apache.org/jira/browse/BEAM-10556)
+  "SameNameButDifferent"
```

Comment:
Here the conflict is because almost every test has a local Inner Class `MockFn` with the same name, I can't figure out a logical way to refactor this.

Reply:
Ok, that soundsd good. Thank you!